sfm/sfm/settings/docker_settings.py

```diff
@@ -12,4 +12,7 @@
 # A hashed version of `SITE_SUPERUSER_PASSWORD` will be store in superuser's `password` field.
 SITE_SUPERUSER_PASSWORD = env.get('SFM_SITE_ADMIN_PASSWORD', 'password')
 
-STATIC_ROOT = "/opt/sfm-static"
\ No newline at end of file
+STATIC_ROOT = "/opt/sfm-static"
+
+MQ_ENV_RABBITMQ_DEFAULT_USER = env.get('MQ_ENV_RABBITMQ_DEFAULT_USER')
```

Comment:
I've suggest that:
* RABBITMQ_USER and RABBITMQ_PASSWORD for the  variable names.
* common.py should have a default value for RABBITMQ_USER and RABBITMQ_PASSWORD (e.g., env.get('SFM_RABBITMQ_USER')) that is overridden by docker_settings.py.